Python/Product/PythonTools/PythonTools.csproj

```diff
@@ -271,9 +271,6 @@
     <Compile Include="..\Analysis\IEnumerableExtensions.cs">
       <Link>IEnumerableExtensions.cs</Link>
     </Compile>
-    <Compile Include="..\Analysis\Interpreter\PythonInterpreterFactoryExtensions.cs">

```

Comment:
This file still exists.  Any reason to stop compiling it within PythonTools.csproj?